packages/amplication-kafka/src/utils/environmentVariables.ts

```diff
@@ -0,0 +1,24 @@
+import { config } from "dotenv";
+import assert from "assert";
+
+config();
```

Comment:
EnvironmentVariables can become singleton and config() can be part of the private contractor

Reply:
Can you extend more?

Reply:
watching 👀

Reply:
@abrl91 we discussed it offline and @tupe12334  changed it to be a singleton and moved the `config()` inside the singleton constructor. after further talking the `dotenv` `config()` is not supposed to be part of this implementation at all. loading the `.env` file should happen one time globally and this should not be kafka library concern from where the environments arrived.